### DISCUSSION TITLE
Option only for daily mail

### DISCUSSION THREAD

**Discussion by LaOberon** [martomi/chiadog#263] [Q&A]
Is there an option in the configuration file to activate only the daily mail?
![image](https://user-images.githubusercontent.com/87464104/125737019-c4711069-8517-42b3-9e9d-e0dcafca44bf.png)

**Comment by mressler** [comment]
Jumping in on this one to request finer grained control over what notifications are emitted. I am inundated with proofs being found to the point where the whole thing is too noisy. How do I turn on or off just one message?

**Reply by martomi** [reply]
See: https://github.com/martomi/chiadog/issues/252
Solution got merged into dev branch yesterday and I'll release it to main within next day.

**Reply by mressler** [reply]
Fantastic, thanks!
